README.md

```diff
@@ -142,6 +142,8 @@ ActiveReporting::Configuration.setting = value
 
 `metric_lookup_class` - The name of a constant used to lookup prebuilt `Reporting::Metric` objects by name. The constant should define a class method called `#lookup` which can take a string or symbol of the metric name. (Default: `::Metric`)
 
+`db_adapter` - Each database has a diferent way of doing the same thing on SQL. This is an abstraction layer on each database. For example `PostGreSQL` have a native `date_trunc` while `MySQL` doesn't so we do the same in another way. Make sure to chosse your database's adapter (Default: `:sqlite3`). If you want MySQL put `:mysql2` and if you want PostGreSQL put `:postgresql`.
```

Comment:
Misspelled "choose" here. Also, I've only known the database to be spelled "PostgreSQL". I'm not sure why the "G" is capitalized

Reply:
Thanks for the corrections, I'm not an English speaker at all :)